### DISCUSSION TITLE
Examples: Tensorboard.

### DISCUSSION THREAD

**Discussion by leonelmaia** [BindsNET/bindsnet#463] [Q&A]
I am new to SNN and when I run the tensorboard example it does not work properly. This is a know issue or am I doing something wrong?

```
# Create the datasets and loaders
# This is dynamic so you can test each dataset easily
dataset_type = getattr(bindsnet.datasets, dataset[3])
dataset_path = os.path.join("..", "..", "data", dataset[3])
train_dataset = dataset_type(
    PoissonEncoder(time=time, dt=dt),
    NullEncoder(),
    root = dataset_path,
    download=True,
    train=True,
    transform=transforms.Compose(
        [transforms.ToTensor(), transforms.Lambda(lambda x: x * 128.0)]
    )
)

```

This is where the first problem appears. The function does not return a dataset and then I cannot get the shape of the first sample.

**Comment by Hananel-Hazan** [comment]
I am not sure what error are you getting, the tensorboard example runs without any issue on my side
![Screenshot 2021-03-15 191323](https://user-images.githubusercontent.com/3954715/111233041-22ee1c80-85e4-11eb-91ed-1142518b93d2.png)
